test/syscall_test/ltp/Makefile

```diff
@@ -0,0 +1,49 @@
+# SPDX-License-Identifier: MPL-2.0
+
+MKFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
+CUR_DIR := $(patsubst %/,%,$(dir $(MKFILE_PATH)))
+INITRAMFS ?= $(CUR_DIR)/../../build/initramfs
+TARGET_DIR := $(INITRAMFS)/opt/ltp
+RUN_BASH := $(CUR_DIR)/run_ltp_test.sh
+WHITELIST := $(CUR_DIR)/whitelist.txt
+EXT2_BLOCKLIST := $(CUR_DIR)/blocklist.ext2
+EXFAT_BLOCKLIST := $(CUR_DIR)/blocklist.exfat
```

Comment:
Having both a whitelist and blocklists is a bit of confusing. I suggest adding a new directory called `test_list` and move them into this directory.

```
ltp/
    test_list/
        all.txt
        blocked/
            ext2.txt
            exfat.txt
```

This new file organization makes their meanings more clear.